wandb/sdk/launch/launch_add.py

```diff
@@ -17,7 +17,8 @@ def push_to_queue(api: Api, queue: str, launch_spec: Dict[str, Any]) -> Any:
 
 
 def launch_add(
-    uri: str,
+    uri: Optional[str] = None,
```

Comment:
You could just simplify this passthrough func and make it just pass forward `*args, **kwargs`? If they are named kwargs then you don't even need to worry about order.

Reply:
We should add documentation to this and leave this as is, in my opinion.